pkgs/os-specific/linux/cpuset/default.nix

```diff
@@ -1,27 +1,37 @@
 { stdenv
 , fetchFromGitHub
-, python2Packages
+, pythonPackages
 }:
 
-python2Packages.buildPythonApplication rec {
+pythonPackages.buildPythonApplication rec {
   pname = "cpuset";
-  version = "1.5.8";
+  version = "1.6";
 
-  propagatedBuildInputs = [ ];
+  propagatedBuildInputs = with pythonPackages; [
+    configparser
+    future
+  ];
+
+  patches = [ ./fix-parenthesis.patch ];
```

Comment:
Please use fetchPatch with ``https://github.com/lpechacek/cpuset/commit/f6cb3eb653d009aa44b719ac21729c9eec8e5ae9.patch``.

Reply:
@SuperSandro2000 This is based on a fork of mine that I may delete in the future (I was planning to take the ownership of this project but I am not so sure nowadays). So I don't think it is a good idea to use `fetchPatch` in this case since this commit may be garbage collected in the future.

Reply:
If you want I can point to this PR instead: https://github.com/lpechacek/cpuset/pull/36

WDYT @SuperSandro2000 ?

Reply:
Either commit is fine for me.

Reply:
Done.